File: ProjetoOficina/ProjetoOficina/Services/UsuarioService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using ProjetoOficina.Models;
using ProjetoOficina.Repository;

namespace ProjetoOficina.Services
{
    public class UsuarioService
    {
        
        public Usuario Login(Usuario usuario)
        {
            Usuarios usuarios = new Usuarios();
            return usuarios.Login(usuario);
        }

        public void Cadastrar(Usuario usuario)
        {
            Usuarios usuarios = new Usuarios();
            usuarios.Cadastrar(usuario);
        }
    }
}

File: ProjetoOficinaFront/src/app/services/usuario.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpHeaders} from '@angular/common/http';
import { Usuario } from '../models/usuario.model';
import { API } from './projetooficina.api';

@Injectable()
export class UsuarioService {

  

  constructor(
    private http: HttpClient
  ) { }

  login(usuario: Usuario){
    return this.http.post(API+'/api/login', usuario);
  }

  buscarPorId(id: number){
    return this.http.post(API+'/api/bucarPorId', id);
  }

  cadastrar(usuario: Usuario){
    return this.http.post(API+'/api/cadastrar/usuario', usuario);
  }
}